datatypes_easyjson.go

```diff
@@ -0,0 +1,471 @@
+// Code generated by easyjson for marshaling/unmarshaling. DO NOT EDIT.
```

Comment:
Need to add a generate command into Makefile or shell script (`generate_mocks.sh` e.g.)

Reply:
Added to makefile